app/src/main/java/one/mixin/android/ui/conversation/ConversationFragment.kt

```diff
@@ -3039,4 +3036,12 @@ class ConversationFragment() :
             closeTool()
         }
     }
+
+    private fun checkAppCardForward(messageItem: MessageItem) {
+        if (messageItem.type == MessageCategory.APP_CARD.name) {
```

Comment:
Why not use `&&` ?

Reply:
this redundant condition has been removed